activerecord/lib/active_record/railties/databases.rake

```diff
@@ -386,32 +386,32 @@ db_namespace = namespace :db do
     task all: ["db:create", :environment, "db:schema:load", :seed]
 
     ActiveRecord::Tasks::DatabaseTasks.for_each(databases) do |name|
-      desc "Creates the #{name} database, loads the schema, and initializes with the seed data (use db:reset:#{name} to also drop the database first)"
+      desc "Create the #{name} database, load the schema, and initialize with the seed data (use db:reset:#{name} to also drop the database first)"
       task name => ["db:create:#{name}", :environment, "db:schema:load:#{name}", "db:seed"]
     end
   end
 
-  desc "Creates all databases, loads all schemas, and initializes with the seed data (use db:reset to also drop all databases first)"
+  desc "Create all databases, load all schemas, and initialize with the seed data (use db:reset to also drop all databases first)"
   task setup: ["db:create", :environment, "db:schema:load", :seed]
 
-  desc "Runs setup if database does not exist, or runs migrations if it does"
+  desc "Run setup if database does not exist, or run migrations if it does"
   task prepare: :load_config do
     ActiveRecord::Tasks::DatabaseTasks.prepare_all
   end
 
-  desc "Loads the seed data from db/seeds.rb"
+  desc "Load the seed data from db/seeds.rb"
   task seed: :load_config do
     db_namespace["abort_if_pending_migrations"].invoke
     ActiveRecord::Tasks::DatabaseTasks.load_seed
   end
 
   namespace :seed do
-    desc "Truncates tables of each database for current environment and loads the seeds"
+    desc "Truncate tables of each database for current environment and load the seeds"
     task replant: [:load_config, :truncate_all, :seed]
   end
 
   namespace :fixtures do
-    desc "Loads fixtures into the current environment's database. Load specific fixtures using FIXTURES=x,y. Load from subdirectory in test/fixtures using FIXTURES_DIR=z. Specify an alternative path (e.g. spec/fixtures) using FIXTURES_PATH=spec/fixtures."
+    desc "Load fixtures into the current environment's database. Load specific fixtures using FIXTURES=x,y. Load from subdirectory in test/fixtures using FIXTURES_DIR=z. Specify an alternative path (e.g. spec/fixtures) using FIXTURES_PATH=spec/fixtures."
```

Comment:
We might need to tweak this to indicate the latter sentences are instructive instead of descriptive:

```suggestion
    desc "Load fixtures into the current environment's database. To load specific fixtures, use FIXTURES=x,y. To load from subdirectory in test/fixtures, use FIXTURES_DIR=z. To specify an alternative path (e.g. spec/fixtures), use FIXTURES_PATH=spec/fixtures."
```

Reply:
Thanks for catching this!